server/services/mongodb-remote/src/test/java/com/mongodb/stitch/server/services/mongodb/remote/internal/SyncMongoClientIntTests.kt

```diff
@@ -899,6 +902,27 @@ class SyncMongoClientIntTests : BaseStitchServerIntTest() {
         }
     }
 
+    @Test
+    fun testConfigure() {
+        val testSync = getTestSync()
+
+        (mongoClient as RemoteMongoClientImpl).dataSynchronizer.enableSyncThread()
+        assertFalse((mongoClient as RemoteMongoClientImpl).dataSynchronizer.isRunning)
+
+        val docToInsert = Document("hello", "world")
+        testSync.insertOneAndSync(docToInsert)
+
+        assertFalse((mongoClient as RemoteMongoClientImpl).dataSynchronizer.isRunning)
```

Comment:
[test] Instead of needing an isRunning as var and exposed, you can instead just add a semaphore to watch a change stream come in.

Reply:
👍